project_stage/views/project_stage_views.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+     <record id="project_stage_form" model="ir.ui.view">
+        <field name="name">project.stage.form</field>
+        <field name="model">project.stage</field>
+        <field name="arch" type="xml">
+            <form string="Task Stage">
+                <group>
+                    <group>
+                        <field name="name"/>
+                        <field name="sequence"/>
+                    </group>
+                    <group>
+                        <field name="case_default"/>
+                        <field name="fold"/>
+                    </group>
+                </group>
+                <field name="description" placeholder="Add a description..."/>
+            </form>
+        </field>
+    </record>
+    <record id="action_project_stage" model="ir.actions.act_window">
+        <field name="name">Stages</field>
```

Comment:
> +        <field name="name">Stages</field>


I think that should be Project Stages in order to do not confuse the menu with the Task Stages.